plugins/modules/win_get_url.ps1

```diff
@@ -144,14 +146,30 @@ Function Get-Checksum {
     }
     return $hash
 }
+Function Set-FileZoneStream {
+    <#
+    .SYNOPSIS
+    Set the input file with Zone.Identifier stream as downloaded from a browser.
+    #>
+    param(
+        [Parameter(Mandatory = $true)][Uri]$Url,
+        [Parameter(Mandatory = $true)][String]$Dest
+    )
+    if (-not (Test-Path -LiteralPath $Dest)) {
+        $module.FailJson("Cannt find '$Dest' for setting stream.")
+    }
+    $stream_dest = $Dest + ":Zone.Identifier"
+    Set-Content -Path $stream_dest -Value '[ZoneTransfer]', 'ZoneId=3', "ReferrerUrl=$Url", "HostUrl=$Url"
```

Comment:
Does a stream work with a `-LiteralPath` parameter? If so we will want to use that here.